src/pages/Governance/constants.tsx

```diff
@@ -12,6 +12,8 @@ export const negativeColorOnHover: string = "#dd6767";
 export const negativeColorWithOpacity: string = "rgba(249,115,115,0.1)";
 // yellow
 export const warningColor: string = "#D8BF45"; // yellow
+// grey
+export const secondaryColor: string = grey[400]; // grey
```

Comment:
should we support theme mode? 
````${theme.palette.mode === "dark" ? grey[100] : grey[400]}````

Reply:
I tried it on light mode too, grey[400] works good on both themes, so I might just keep it.